examples/run-blockchain/index.ts

```diff
@@ -11,6 +11,15 @@ const BlockHeader = require('ethereumjs-block/header.js')
 const testData = require('./test-data')
 const level = require('level')
 
+interface IBlockchainHead {
```

Comment:
Did you get an error without this interface? For reference, `ethereumjs-block` has been migrated to typescript, but is pending a release https://github.com/ethereumjs/ethereumjs-block/pull/74. Afterwards we could just use the `Block` type.

Reply:
Yes, without the interface I get this error:

```
/home/hugo/.nvm/versions/node/v10.16.0/lib/node_modules/ts-node/src/index.ts:245
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
index.ts:58:50 - error TS2339: Property 'header' does not exist on type '{}'.

58   console.log('New head:', '0x' + blockchainHead.header.hash().toString('hex'))
                                                    ~~~~~~
```

`blockchainHead.hash()` doesn't work either:

```
TSError: ⨯ Unable to compile TypeScript:
index.ts:58:50 - error TS2339: Property 'hash' does not exist on type '{}'.

58   console.log('New head:', '0x' + blockchainHead.hash().toString('hex'))
```